config/configuration.yaml.example

```diff
@@ -102,3 +102,15 @@ automation 2:
 
   execute_service: notify.notify
   service_data: {"message":"It's 4, time for beer!"}
+
+sensor:
+  platform: systemmonitor
+  devices:
+    - {type: 'disk_use_percent', arg: '/'}
```

Comment:
I see that this works in YAML but it doesn't feel very YAMLy to me. I would prefer this:

```yaml
sensor:
  platform: systemmonitor
  devices:
    - type: disk_use_percent
      arg: /
    - type: disk_use_percent
      arg: /home
    - type: disk_use
      arg: /home
    - type: disk_free
      arg: /
    - type: memory_use_percent
    - type: memory_use
    - type: memory_free
    - type: processor_use
```